ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -119,7 +119,7 @@ def initialize(self: Analysis) -> None:
 
         # generate ensemble da YAML file
         logger.debug(f"Generate ensemble da YAML file: {self.task_config.fv3jedi_yaml}")
-        ensda_yaml = parse_j2yaml(self.task_config.ATMENSYAML, self.task_config)
+        ensda_yaml = parse_j2yaml(self.task_config.ATMENSYAML, self.task_config, searchpath=self.gdasapp_j2tmpl_dir)
```

Comment:
replace `ATMENSYAML` with `JEDIYAML` here if this variable is renamed in `config.atmanl`
```suggestion
        ensda_yaml = parse_j2yaml(self.task_config.JEDIYAML, self.task_config, searchpath=self.gdasapp_j2tmpl_dir)
```

Reply:
Change committed at ed042392.